develop/dev-guide-proxysql-integration.md

```diff
@@ -1,682 +1,1115 @@
+
+### Prerequisites
+
+Depending on the option you choose, you might proceed with downloading and installing the following packages:
+
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [Docker](https://docs.docker.com/get-docker/)
+- [Python3](https://www.python.org/downloads/)
+- [Docker Compose](https://docker-docs.netlify.app/compose/install/)
+- [MySQL client](https://dev.mysql.com/doc/refman/8.0/en/mysql.html)
+
+You can follow the installation instructions as below:
+
+<SimpleTab groupId="os">
+
+<div label="macOS" value="macOS">
+
+1. [Download]((https://docs.docker.com/get-docker/)) and start `Docker` (the Docker desktop already includes the docker-compose).
+2. Run the following command to install `python` and `mysql-client`:
+
+    ```bash
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+    brew install python mysql-client
+    ```
+
+</div>
+
+<div label="CentOS" value="CentOS">
+
+```bash
+curl -fsSL https://get.docker.com | bash -s docker
+yum install -y git python39 docker-ce docker-ce-cli containerd.io docker-compose-plugin mysql
+systemctl start docker
+```
+
+</div>
+
+<div label="Windows" value="Windows">
+
+1. Download and install Git.
 
-### Test environment
+    1. Download the **64-bit Git for Windows Setup** package from the [Git Windows Download](https://git-scm.com/download/win) page.
+    2. Install the Git package by following the setup wizard. You can click **Next** for a few times to use the default installation settings.
 
-<SimpleTab groupId="startup-tidb">
+        ![proxysql-windows-git-install](/media/develop/proxysql-windows-git-install.png)
 
-<div label="TiDB Cloud" value="tidb-cloud">
+2. Download and install MySQL Shell.
+
+    1. Download the ZIP file of MySQL Installer from the [MySQL Community Server Download](https://dev.mysql.com/downloads/mysql/) page.
+    2. Unzip the file, and locate `mysql.exe` in the bin folder. You need to add the path of the bin folder to the System Variable and set it into the `Path` variable at Git Bush:
+
+        ```bash
+        echo 'export PATH="(your bin folder)":$PATH' >>~/.bash_profile
+        source ~/.bash_profile
+        ```
 
-You can refer to [Build a TiDB cluster in TiDB Cloud (Serverless Tier)](/develop/dev-guide-build-cluster-in-cloud.md).
+        For example:
+
+        ```bash
+        echo 'export PATH="/c/Program Files (x86)/mysql-8.0.31-winx64/bin":$PATH' >>~/.bash_profile
+        source ~/.bash_profile
+        ```
+
+3. Download and install Docker.
+
+    1. Download Docker Deskstop Installer from the [Docker Download](https://www.docker.com/products/docker-desktop/) page.
+    2. Double-click the installer to run it. After the installation is completed, you will be prompted for a restart.
+
+        ![proxysql-windows-docker-install](/media/develop/proxysql-windows-docker-install.png)
```

Comment:
```suggestion
```

Reply:
@Icemap  Do you think we can remove this screenshot because the version (4.14.1) there is easy to be outdated sometime?

Reply:
I'll pixelate the version part.

Reply:
Done